src/main/java/com/checkmarx/flow/service/GitHubService.java

```diff
@@ -91,17 +91,20 @@ void sendMergeComment(ScanRequest request, String comment){
 
     void startBlockMerge(ScanRequest request, String url){
         if(properties.isBlockMerge()) {
+            final String PULL_REQUEST_STATUS = "pending";
             HttpEntity<?> httpEntity = new HttpEntity<>(
-                    getJSONStatus("pending", url, "Checkmarx Scan Initiated").toString(),
+                    getJSONStatus(PULL_REQUEST_STATUS, url, "Checkmarx Scan Initiated").toString(),
                     createAuthHeaders()
             );
```

Comment:
Can we set a timeout for the 'pending' step?
what will happen if CxFlow stopped while the PR is pending?

Reply:
Regarding the timeout, I guess you are asking if github can automatically change the status from 'pending' to something else after a timeout? I don't see such functionality in the API docs.

If cxflow stops while the PR is pending, user needs to restart cxflow and make some change in the pull request, e.g. close/reopen.